src/lib/logMsg/logMsg.cpp

```diff
@@ -971,7 +971,7 @@ static char* timeGet(int index, char* line, int lineSize)
 }
 
 
-
+#if 0
```

Comment:
I removed the old LM_S ('S' standing for timeStamp) to make room for a new LM_S ('S' standing for Summary), as the broker doesn't use LM_S anyway.

However, for the future of the logMsg library, I prefer to keep the function timeStampGet, thus the #if 0/#endif